workspace/2023-01-conda-January-release.md

```diff
@@ -0,0 +1,18 @@
+# Discourse
+
+The first conda release of 2023 is now available! To update run:
+
+```
+conda install -n base conda=23.1.0
+```
+
+This was a smaller impact release given the winter holidays so most of the changes focus on bug fixes.
+
+Check out the [changelog](https://github.com/conda/conda/releases/tag/23.1.0) for the complete list of improvements.
+
+
+# Twitter
+
+The first #conda release of 2023 is here!
+
+<Discourse link>
```

Comment:
Not sure how we'll add this, @tnabtaf any ideas?

Reply:
@jezdez I can putting it in [News](https://conda.discourse.group/c/news/6) or [Users ⇒ conda](https://conda.discourse.group/c/use/use-conda/10).

Or, are you asking a different question?